src/css/vars.css

```diff
@@ -1,10 +1,15 @@
 :root {
   /* colors */
-  --color-blue-white-gradient: linear-gradient(rgb(36, 36, 167) 0%, rgb(9, 141, 222) 100%);
+  --color-dark-blue: #070766;
+  --color-light-blue: #00BAFF;
+  --color-blue: #173CFF;
+
+  --color-blue-white-gradient: var(--color-dark-blue);
```

Comment:
I think this is superseding the defined gradient as it shows up a solid color on my screen